Pearson-Advance/openedx-themes
# Improving Pearson default theme

This PR aims to make some improvements in the pearson default theme to make it consistent with the MFEs.

## Changes

- [x] Adjust header and footer styles and structure.
- [x] Make show programs link configurable from site configurations.

## Screenshots
![Screenshot from 2023-06-30 16-46-55](https://github.com/Pearson-Advance/openedx-themes/assets/62396424/668eee3f-cc9b-49bc-9296-b55b97b2d656)
![Screenshot from 2023-06-30 16-46-50](https://github.com/Pearson-Advance/openedx-themes/assets/62396424/b8889b3a-6adb-4868-9d32-e647e9e8e635)

## How to test

- Checkout to this branch
- Enable comprehensive theming
- Add pearson-theme to a site theme
- Check that the changes are applied correctly

## Reviewers
